scenes/Pilot/Pilot.gd

```diff
@@ -0,0 +1,14 @@
+extends Node2D
+
+
+var player_character = null
```

Comment:
Should we set some default values for these? In case we try to reference them before they get set. For example if we do $pilot.ship.global_position before a ship is set the game may crash.

Reply:
I was thinking we would never have a pilot without a ship. One limiting factor is that all properties of the ship node and its children could be accessed, including default ones. I don't think it's practical to set defaults for every property in case we forget to instance in a ship scene. We could set defaults just for the properties we are actually looking up but the game is still broken without a ship even if we get no crash.

Reply:
I added this to the trello board as a potential future improvement, fyi. "consider refactoring world script (ship assignment) code into a function housed in pilot.gd, the dialogical singal in world then calls down to that function. Would make it so we could assign ships in other scenes, if necessary." We can discuss that in class today too.